integration/helpers.go

```diff
@@ -63,6 +63,11 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	Loopback = "127.0.0.1"
+	Host     = "localhost"
+)
+
```

Comment:
What is the purpose of relocating these values from `integration_test.go` to here?